src/components/NewCourse.js

```diff
@@ -104,6 +110,48 @@ class NewCourse extends React.Component {
             </FormText>
           </Col>
         </FormGroup>
+        <FormGroup row>
```

Comment:
<img width="721" alt="Screen Shot 2019-11-02 at 4 19 58 AM" src="https://user-images.githubusercontent.com/1476544/68070187-50f0fa80-fd28-11e9-9707-b04089d9246a.png">

This looks off on medium-width displays. Could you aim for reflow similar to the text inputs? You can see a good example of how switches should look at a medium width by looking at the queue creation panel.